File: Beecrowd/1042 Sort Simples/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int x, y, z, menor = 0, meio = 0, maior = 0;

        x = sc.nextInt();
        y = sc.nextInt();
        z = sc.nextInt();

        if (x < y && x < z)
            menor = x;
        else if (x > y && x > z)
            maior = x;
        else
            meio = x;

        if (y < x && y < z)
            menor = y;
        else if (y > x && y > z)
            maior = y;
        else
            meio = y;

        if (z < x && z < y)
            menor = z;
        else if (z > x && z > y)
            maior = z;
        else
            meio = z;

        System.out.println(menor + "\n" + meio + "\n" + maior + "\n");
        System.out.println(x + "\n" + y + "\n" + z);

        sc.close();
    }

}

File: Beecrowd/1042 Sort Simples/main.cpp
#include <iostream>

using namespace std;

int main() {

    int X, Y, Z, menor, meio, maior;

    cin >> X;
    cin >> Y;
    cin >> Z;

    if (X < Y && X < Z)
        menor = X;
    else if (X > Y && X > Z)
        maior = X;
    else
        meio = X;
    if (Y < X && Y < Z)
        menor = Y;
    else if (Y > X && Y > Z)
        maior = Y;
    else
        meio = Y;
    if (Z < X && Z < Y)
        menor = Z;
    else if (Z > X && Z > Y)
        maior = Z;
    else
        meio = Z;

    cout << menor << endl;
    cout << meio << endl;
    cout << maior << endl << endl;
    cout << X << endl;
    cout << Y << endl;
    cout << Z << endl;

    return 0;
}